### ISSUE TITLE
Ugly blue border around filter when I click it

### ISSUE THREAD

**Issue by bitboxer** [airyhq/airy#842]
this blue border appears when I click on filter:

<img width="203" alt="Screenshot 2021-02-02 at 16 21 46" src="https://user-images.githubusercontent.com/56195/106621393-cfaf9580-6572-11eb-8393-c83a3392cded.png">

Ideally we should have a focus outline when tabbing to it via keyboard because of accesbility, but when clicking it should not be visible.
